samples/boards/stm32/i2c_config/README.rst

```diff
@@ -0,0 +1,78 @@
+.. _i2c_config:
+
+STM32 i2c_config
+################
```

Comment:
What about making this sample specific to timing V2 only ?
```suggestion
STM32 I2C V2 timings
####################
```

For V1, this sample doesn't add much value and this would simplify the sample configuration a bit.

Reply:
done